packages/zksend/experimental/package.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think you can remove this since the `packages/zksend/experimental/unified-adapter/package.json` file is the correct one